_config/graphql-schema-default.yml

```diff
@@ -0,0 +1,30 @@
+---
+Name: 'graphql-schema-default'
```

Comment:
Why would `silverstripe/admin` need to influence the default schema? You might not even run admin to use GraphQL, right? Shouldn't that either be in the GraphQL module, or in code advising how to set up your own schema?

Reply:
It's always been this way: https://github.com/silverstripe/silverstripe-admin/blob/1/_config/injector.yml

It's a lot of boilerplate to light up a new schema, so the core recipe ships with a prebaked, unrouted schema. "Activating" the default schema is just routing it.

Interesting that it's tied to admin and not graphql, though. I'm not sure what went into that decision, but this is just legacy.

Reply:
The file you linked to (`injector.yml`) only configures the `admin` schema, not the `default` schema. I can't find any relevant [mention](https://github.com/silverstripe/silverstripe-admin/search?l=YAML&q=default) of "default" in the admin module config before this PR.

Reply:
Ah yup, you're right. This has always been in graphql. Needs to be moved.